other/edge-server/client/README.md

```diff
@@ -9,15 +9,16 @@ React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App based on the `edge-server.todo` template app, and
-   get Edge Server enabled for your new app.
-2. Download, configure, and start the Edge Server.
-3. Install dependencies and start the Node.js Express server.
-4. Install dependencies and start the React server.
+1. Create an App Services App based on the `edge-server.todo` template app.
+2. Install Edge Server dependencies
+3. Download, configure, and start the Edge Server.
+4. Install Express server and React client dependencies
+5. Start the Express server and React client.
 
 This is a MERN stack that connects to the Edge Server instead of Atlas.
 In this version of the example app, MongoDB wire protocol connections connect
-directly to Edge Server without authorization.
+directly to Edge Server. You can use email/password authentication with an
```

Comment:
per my previous comment, i think the app config for this should just enable anonymous auth